content/02.main-text.md

```diff
@@ -119,7 +119,7 @@ Going forward, we hope to integrate Manubot with the larger JATS ecosystem.
 For now however, the primary Manubot output is HTML intended to be viewed in a web browser.
 
 Manubot performs continuous publication: every update to a manuscript's source is automatically reflected in the online outputs.
-The approach uses continuous integration (CI) [@doi:10.1038/550143a; @doi:10.1038/nbt.3780], specifically [Travis CI](https://travis-ci.org/) at the moment, to monitor for changes.
+The approach uses continuous integration (CI) [@doi:10.1038/550143a; @doi:10.1038/nbt.3780; @doi:10.1101/344804, @doi:10.1101/344804], specifically [Travis CI](https://travis-ci.org/) at the moment, to monitor for changes.
```

Comment:
Has been added twice.

Reply:
@dhimmel Ah thanks for the catch. Will fix now.